packages/design-mui/src/components/MuiFilter/MuiFilter.tsx

```diff
@@ -33,7 +34,55 @@ interface PopoverProps extends MuiFilterRowProps {
   clearFilter: () => void
 }
 
+interface Option {
+  operator: string
+  text: string
+}
+
+interface OperatorOption {
+  [key: string]: Option[]
+}
+
+const operatorOptions: OperatorOption = {
+  string: [
+    { operator: "ilike", text: "contains" },
+    { operator: "$eq", text: "equals" },
+    { operator: "empty", text: "is empty" },
+    { operator: "nempty", text: "is not empty" },
+  ],
+  date: [
+    { operator: "$eq", text: "is" },
+    { operator: "$gt", text: "is after" },
+    { operator: "$gte", text: "is on or after" },
+    { operator: "$lt", text: "is before" },
+    { operator: "$lte", text: "is on or before" },
+    { operator: "empty", text: "is empty" },
+    { operator: "nempty", text: "is not empty" },
+  ],
+}
+
+//change the operator if the selected operator is not compatible with the new column type
+const checkOperator = (
```

Comment:
if this returns the current (or new) operator, lets rename this function `getOperator` - with check my first assumption is it is returning a boolean